src/biosHandler.cpp

```diff
@@ -45,7 +53,42 @@ void BiosHandler::checkAndListenPldmService()
     if (dbusUtility::isServiceRunning(constants::pldmServiceName))
     {
         l_nameOwnerMatch.reset();
-        // TODO: retore BIOS attribute from here.
+        m_specificBiosHandler->backUpOrRestoreBiosAttributes();
+
+        // Start listener now that we have done the restore.
+        listenBiosAttributes();
     }
 }
+
+template <typename T>
+void BiosHandler<T>::listenBiosAttributes()
+{
+    static std::shared_ptr<sdbusplus::bus::match_t> l_biosMatch =
+        std::make_shared<sdbusplus::bus::match_t>(
+            *m_asioConn,
+            sdbusplus::bus::match::rules::propertiesChanged(
+                constants::biosConfigMgrObjPath,
+                constants::biosConfigMgrService),
+            [this](sdbusplus::message_t& l_msg) {
+        m_specificBiosHandler->biosAttribsCallback(l_msg);
+    });
+}
+
+void IbmBiosHandler::biosAttribsCallback(sdbusplus::message_t& i_msg)
```

Comment:
as we using full 'Attributes' most of the placess can we keep it same here.

Reply:
done.